docs/source/glossary.rst

```diff
@@ -67,14 +67,19 @@ sktime.
         In ``sktime`` reduction is used to allow one learning task to be adapted as a solution for an alternative task.
 
     Variable
-        Refers to some measurement of in terest. Variables may be cross-sectional (e.g. time-invarient measurements like a patient's place of birth) or
+        Refers to some measurement of interest. Variables may be cross-sectional (e.g. time-invariant measurements like a patient's place of birth) or
         :term:`time series`.
 
     Timepoint
-        The point in time that an observation is made. A timee point may represent an exact point in time (a timestamp),
+        The point in time that an observation is made. A time point may represent an exact point in time (a timestamp),
         a timeperiod (e.g. minutes, hours or days), or simply an index indicating the position of an observation in the sequence of values.
 
     Instance
         A member of the set of entities being studied and which an ML practitioner wishes to generalize. For example,
         patients, chemical process runs, machines, countries, etc. May also be referred to as samples, examples, observations or records
         depending on the discipline and context.
+
+    Tabular
+        Is a setting where each :term:`timepoint` of the :term:`univariate time series` being measured for each instance are treated as feature and
+        stored as a primitive data type in the DataFrame’s cells. E.g., there are N instances of time series and each has T timepoints, this would
```

Comment:
Include a cross-reference to the glossary terms "instance" and "timepoints". 

E.g. there are N :term:`instances <instance>` of the time series and each has T :term:`timepoints <timepoint>`